extra/intel-tbb/PKGBUILD

```diff
@@ -0,0 +1,35 @@
+# $Id$
+# Maintainer: Stéphane Gaudreault <stephane@archlinux.org>
+# Contributor: Thomas Dziedzic < gostrc at gmail >
+# Contributor: Denis Martinez <deuns.martinez AT gmail.com>
+
+# ALARM: Romain Reignier <rom.reignier AT gmail.com>
+# - add armv7h architecture
+# - add specfic CXXFLAGS for armv7h
+
+buildarch=4
+
+pkgname=intel-tbb
+pkgver=4.3_20150209
+pkgrel=1
+pkgdesc='High level abstract threading library'
+arch=('i686' 'x86_64')
+url='http://www.threadingbuildingblocks.org/'
+license=('GPL')
+depends=('gcc-libs')
+source=("http://threadingbuildingblocks.org/sites/default/files/software_releases/source/tbb${pkgver/\./}oss_src.tgz")
+sha1sums=('cb17bee2a9c98a2b98f3ff16208c1c1fae29e6ab')
+
+build() {
+  cd tbb${pkgver/\./}oss
+  CXXFLAGS+="-DTBB_USE_GCC_BUILTINS=1 -D__TBB_64BIT_ATOMICS=0"
```

Comment:
The space in my previous comment after the beginning quote wasn't a typo.  You need to separate the addition from the other flags.